infra/charts/feast-feature-server/README.md

```diff
@@ -13,9 +13,12 @@ helm repo update
 
 Install Feast Feature Server on Kubernetes
 
+Feast Feature Server supported with deployment mode using property `feast_mode` - online (default), offline, ui and registry. User need to set feast_mode based on choice of deployment.
+
 A base64 encoded version of the `feature_store.yaml` file is needed. Helm install example:
 ```
-helm install feast-feature-server feast-charts/feast-feature-server --set feature_store_yaml_base64=$(base64 feature_store.yaml)
+helm install feast-feature-server feast-charts/feast-feature-server --set feast_mode=online  --set feature_store_yaml_base64=$(base64 feature_store.yaml)
+helm install feast-offline-server feast-charts/feast-feature-server --set feast_mode=offline  --set feature_store_yaml_base64=$(base64 feature_store.yaml)
```

Comment:
maybe change one of the names here so users don't get a conflict error if they try both commands consecutively.

Reply:
The name are different `helm install feast-feature-server` and `helm install feast-offline-server` is this is your ask ?